docs/validator/run/mainnet.md

```diff
@@ -445,6 +445,33 @@ You can run multiple proxies by deploying additional proxies per the instruction
 
 :::
 
+### Limiting block space for transactions paid in alternative ERC-20 gas currencies
+
+As described in the protocol documentation, Celo allows users to [pay for gas using tokens](/protocol/transaction/erc20-transaction-fees). There is a governable list of accepted tokens. However, the Celo blockchain client starting with version 1.8.1 implements a protective mechanism that allows validators to control the percentage of available block space used by transactions paid with an alternative fee currency (other than CELO) more precisely.
+
+There are two new flags that control this behavior:
+
+1. `celo.feecurrency.limits` with a comma-separated `currency address hash=limit` mappings for currencies listed in the `FeeCurrencyWhitelist` contract, where `limit` represents the maximal fraction of the block gas limit as a float point number available for the given fee currency.
```

Comment:
Maybe add underscores in `currency address hash=limit` toi make clear that one value is expected.

Reply:
Good point, added.